package.json

```diff
@@ -6,75 +6,67 @@
   "module": "index.mjs",
   "main": "index",
   "files": [
+    "src",
```

Comment:
we should separate the `dependencies` and `devDependencies` in this file as was being done in https://github.com/sveltejs/svelte/pull/8523. i'm not sure if you want to try to merge that PR first or copy those changes into this PR

Reply:
If copying over the changes, `source-map` needs to be kept and so does `util` polyfill for the UMD bundle.

Reply:
Thanks for catching that. I updated https://github.com/sveltejs/svelte/pull/8523 to address that